src/main/java/dev/meyi/bn/utilities/Suggester.java

```diff
@@ -68,4 +68,15 @@ public static void basic() {
       e.printStackTrace();
     }
   }
+
+  public static String setSuggestionLength(int Length){
+    if(Length >= BazaarNotifier.bazaarDataRaw.length()-1){
+      return "Error Number to high" + String.valueOf(Length);
+    }else{
+      BazaarNotifier.config.put("FlippingListLength" , Length);
+      return "Seting CraftingListLength to " + String.valueOf(Length);
+    }
+  }
```

Comment:
A few small things to change here:
1. `int Length` should be `int length`
2. Instead of doing `-1` on the length to check, let's just make it `length > BazaarNotifier.bazaarDataRaw.length()`
3. The return message should be changed:
    - The error should be something like `length + " is too long"`
    - The success should be something like `"Item list size set to " + length`